src/org/opendatakit/briefcase/ui/automation/AutomationPanel.java

```diff
@@ -80,6 +127,7 @@ public static AutomationPanel from(Http http, BriefcasePreferences appPreference
     return new AutomationPanel(
         AutomationPanelForm.from(http, forms),
         forms,
+        BriefcasePreferences.forClass(AutomationPanel.class),
```

Comment:
This shouldn't be required on this class since we're removing all interaction with preferences.